### PR TITLE
Fix `storybookUrl` by removing `iframe.html` suffix

### LINKED ISSUE #409: GitHub action output `storybookUrl` includes iframe.html
I'm setting up a PR workflow that publishes to Chromatic and then adds a Deployment using the `storybookUrl` output. But it looks like as of 5.10.0, that URL includes `/iframe.html`, which leads to the isolated view with no parameters.

IMO, it should just point to the root URL for the storybook deployment.

### PR DESCRIPTION
**Pull request by ndelangen**
update the action's storybookUrl output, to NOT include `iframe.html` as requested in #409

### CODE REVIEW

File: action-src/main.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ async function runChromatic(options): Promise<Output> {
     url: ctx.build?.webUrl,
     code: ctx.exitCode,
     buildUrl: ctx.build?.webUrl,
-    storybookUrl: ctx.build?.cachedUrl,
+    storybookUrl: ctx.build?.cachedUrl.replace(/iframe\.html.*$/, ''),
```
Comment (tmeasday) [inline action-src/main.ts:110]: Why the `.*` at the end?

PS we also have a `build.storybookUrl` that we could request, perhaps could be more reliable?

Comment (ndelangen) [reply action-src/main.ts:110]: I added that to also strip off any query params (which would not work for the manager)

Comment (ghengeveld) [reply action-src/main.ts:110]: There should never be any query params on cachedUrl I think, but can't hurt to guard against it I suppose.

**Review by ghengeveld — APPROVED** [review]

**Comment by ghengeveld** [comment]
Released in [6.5.0](https://github.com/chromaui/chromatic-cli/blob/main/CHANGELOG.md#650---2022-02-21).

**Pull request merged**